content/Constants/Constants.q.md

```diff
@@ -0,0 +1,1453 @@
+# Constants
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+class Circle
+{
+    const PI = 3.1416;
+    
+    public static function area($radius)
+    {
+        return self::PI * $radius * $radius;
+    }
+}
+
+echo "The PI value is: " . Circle::PI;
+?>
+```
+/// type=SS, answer=[5]
+
+Execute the program. What is its output?
+
+ - It prints `PI`.
+
+ - It prints `3.1416`.
+
+ - It prints `area()`.
+
+ - It prints `$radius`.
+
+ - It prints `The PI value is: 3.1416`.
+
+
+/// type=SS, answer=[2]
+
+Which of the following is a class?
+
+ - `PI`
+
+ - `Circle`
+
+ - `self::PI`
+
+ - `Circle::PI`
+
+ - `const PI = 3.1416;`
+
+
+/// type=SS, answer=[2]
+
+Which of the following is a static method?
+
+ - `PI`
+
+ - `area()`
+
+ - `Circle`
+
+ - `$radius`
+
+ - `Circle::PI`
+
+
+/// type=SS, answer=[3]
+
+In the statement `const PI = 3.1416;` on line 4, what is `const`?
+
+ - It is a class.
+
+ - It is a method.
+
+ - It is a keyword.
+
+ - It is a constant.
+
+ - It is a property.
+
+
+/// type=SS, answer=[5]
+
+In the statement `const PI = 3.1416;` on line 4, what is `PI`?
```

Comment:
Grammarly suggested this correction:

statement,

"It appears that you are missing a comma after the introductory phrase **In the statement**.
Consider adding a comma."